src/components/LoanCards/Buttons/LendCtaExp.vue

```diff
@@ -88,13 +88,18 @@
 			Checkout now
 		</kv-ui-button>
 
-		<!-- Funded, refunded, expired/ allSharesReserved button -->
+		<!-- Refunded, allSharesReserved button -->
 		<kv-ui-button
 			v-if="showNonActionableLoanButton"
 			class="tw-inline-flex tw-flex-1"
 		>
 			{{ ctaButtonText }}
 		</kv-ui-button>
+
+		<!-- Funded / expired -->
+		<div v-if="isFunded" class="tw-w-full tw-text-center tw-rounded tw-bg-tertiary tw-p-2">
```

Comment:
Background should be the same as the loan callouts, it's darker currently.

Reply:
oh, having double bg https://github.com/kiva/ui/blob/505fad3374a8f83a8ef491b3fc997a76c4f44fe0/src/components/LoanCards/LoanTags/LoanCallouts.vue#L20

Reply:
Oh yeah, I think we want `background: #f1f1f1;`